bobtimus/src/loan.rs

```diff
@@ -62,12 +63,11 @@ pub struct Term {
     pub interest_mod: Decimal,
 }
 
-/// Allows to specify a better rate for users that
+/// LTV (Loan-to-value) ratio defines the relation of loan value to collateral value, i.e.
+/// (principal_amount / (collateral_amount * current_rate)
 #[derive(Debug, Clone, Copy, serde::Serialize)]
-pub struct Collateralization {
-    pub collateralization: Decimal,
-    /// Interest to be added on top of the base interest rate for this term.
-    pub interest_mod: Decimal,
+pub struct Ltv {
+    pub ltv: Decimal,
 }
 
 // TODO: Make sure that removing sat_per_vbyte is OK here
```

Comment:
sidenote: this comment is outdated and can be removed (sry, I forgot to remove it)